### PR TITLE
Feature/rickwoltheus/documents list

### PR DESCRIPTION
**Pull request by RickWoltheus**
This pr adds the following
- documents upload modal + button container 
- documents delete modal + button container
- documents list
- ParticipantsTabs component

### CODE REVIEW

File: client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        setFile(undefined)
+
+        if (fileRef && fileRef.files) {
+            fileRef.files = new DataTransfer().files
+        }
+    }
+
+    async function handleUpload(e: React.FormEvent<HTMLFormElement>) {
+        e.preventDefault()
+        const formData = Forms.getFormDataFromFormEvent<{ fileUpload: File }>(e)
```
Comment (bbawuah) [inline client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx:125]: Will it also be possible to upload multiple documents at the same time?

Comment (RickWoltheus) [reply client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx:125]: No it's not, we discussed with Jesse that it should be single file only

Comment (RickWoltheus) [reply client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx:125]: I will double check it though

File: client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Documents/ParticipantsDocumentsView.tsx [hunk 2]
```diff
@@ -1,19 +1,73 @@
...
     const { i18n } = useLingui()
+    const { data, loading, error } = useMockQuery([
+        {
+            id: 'my id',
+            fileName: 'bestand.pdf',
+            createdAt: new Date().toString(),
+            filePath: 'https://file-examples-com.github.io/uploads/2017/10/file-sample_150kB.pdf',
+        },
+    ])
+
```
Comment (bbawuah) [inline client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Documents/ParticipantsDocumentsView.tsx:37]: nice mock solution 👍🏿

**Review by bbawuah — APPROVED** [review]

File: client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx (hunk 1 above)
Comment (RickWoltheus) [reply client/src/components/Domain/Documents/Modals/DocumentUploadModal.tsx:125]: Checked it with Jesse, it shouldn't be possible to upload more then one file

**Pull request merged**